src/writer/blocks/writerchatreplywithtoolconfig.py

```
@@ -0,0 +1,399 @@
+            if not org_id:
+                raise ValueError("Organization ID not found. Set x-organization-id header or WRITER_ORG_ID environment variable.")
+            
+            if not user_id:
+                user_id = os.getenv("WRITER_USER_ID", "1")
+            
+            try:
+                user_id = int(user_id)
+            except (ValueError, TypeError):
+                user_id = 1
+            
+            url = f"http://localhost:8001/api/v1/namespaces/default/services/mcp-gateway:80/proxy/api/private/mcp-gateway/organization/1/mcp"
```

Ruff (0.14.3)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)